.github/workflows/bindings-wasm.yml

```diff
@@ -86,16 +86,16 @@ jobs:
         if: ${{ startsWith(matrix.os, 'ubuntu') }}
         uses: "./.github/actions/private-tangle/setup"
 
-      # - name: Run Yarn Test
-      #   if: ${{ startsWith(matrix.os, 'ubuntu') }}
-      #   working-directory: bindings/wasm/
-      #   run: yarn test
+      - name: Run Yarn Test
+        if: ${{ startsWith(matrix.os, 'ubuntu') }}
+        working-directory: bindings/wasm/
+        run: yarn test
 
       - name: Run Yarn Test
         if: ${{ !startsWith(matrix.os, 'ubuntu') }}
         working-directory: bindings/wasm/
         # Only run test that doesn't require a node because we can't use the private tangle script on macos and windows
-        run: yarn test -t 'Wallet methods'
```

Comment:
wallet methods previously didn't require a node?

Reply:
There is just nothing called `Wallet methods`

Reply:
Alright